tracing-attributes/src/lib.rs

```diff
@@ -227,10 +227,20 @@ mod expand;
 ///     42
 /// }
 /// ```
-/// The return value event will have the same level as the span generated by `#[instrument]`.
-/// By default, this will be `TRACE`, but if the level is overridden, the event will be at the same
+/// The return value event will by default have the same level as the span generated by `#[instrument]`.
+/// By default, this will be `TRACE`, but if the span level is overridden, the event will be at the same
 /// level.
 ///
+/// It's also possible to override the level for the `ret` independently:
```

Comment:
i think this reads better with the word "event" added
```suggestion
/// It's also possible to override the level for the `ret` event independently:
```